docs/UNLOCK_MECHANISM.md

```diff
@@ -0,0 +1,35 @@
+# Unlock Mechanism
+
+This document describes what happens with the app and the lib during the wallet start mechanism after an unlock
+
+1. App is opened
+1. Default screen, as defined on the `AppStack` on `src/App.js` is `PinScreen`, so the Pin is requested from the user
+1. With the PIN, the words are retrieved and stored on the `initWallet` redux store item using the `setIinitWallet` action
+1. The app navigates to `LoadHistoryScreen`, which retrieves the words from the redux from the previous step and dispatches `startWalletRequested`
+1. `startWallet` saga is called and decides whether to load using the wallet-service facade or the old facade. We know that the error is happening in the fullnode facade, so this will ignore the wallet-service specific parts
```

Comment:
Which error are we talking about?

Reply:
Oops, this was copied from [this issue](https://github.com/HathorNetwork/internal-issues/issues/144), so that was the error I was mentioning here

Fixed in 712f2c1fd705589f60c8a61611180dc4e85f85a0